src/components/export/ExportDialog.tsx

```diff
@@ -0,0 +1,196 @@
+import React, { useState } from 'react';
+import {
+  IonContent,
+  IonHeader,
+  IonTitle,
+  IonToolbar,
+  IonList,
+  IonItem,
+  IonLabel,
+  IonCheckbox,
+  IonRadioGroup,
+  IonRadio,
+  IonButton,
+  IonButtons,
+  IonIcon,
+  IonToast,
+  IonCard,
+  IonCardContent,
+  IonText,
+  IonLoading,
+} from '@ionic/react';
+import { close, downloadOutline } from 'ionicons/icons';
+import { useExportData } from '@/hooks/queries/useDataManagementQueries';
+import { ExportFormat } from '@/services/DataManagementService';
+import { School } from '@/db/entities';
+import { useUserName } from '@/hooks/queries';
+
+interface ExportDialogProps {
+  isOpen: boolean;
+  onClose: () => void;
+  school: School;
+}
```

Comment:
```suggestion
interface ExportDialogProps {
  isOpen: boolean;
  onClose: () => void;
}
```

This should not depend on `School`.